### ISSUE TITLE
Let web UI  communicate directly with the Management API

### ISSUE THREAD

**Issue by snizhana-dynnyk** [giantswarm/roadmap#310]
## User Story

_- As a user, I want Happa talk to Management API so that I can access functionality in Management API._

## Details, Background

All functionality Happa has to be migrated to talk directly to Management API instead of rest API. All existing functionality in `gsctl` has to be moved to `kubectl gs`. 
We should avoid/prevent adding any new functionality to rest API as it is officially deprecated.

## Blocked by / depends on

- [ ] Move all customers to use SSO in happa
- [ ] #116 Add support for generating certificates to access workload clusters via the Management API
- [ ] #103 Make cluster/machinedeployment/appcatalogs CRs namespaced by organization

### TODO

- [ ] Migrate Apps to MAPI
- [ ] Migrate Azure cluster management to MAPI
- [ ] Migrate AWS cluster management to MAPI

**Comment by snizhana-dynnyk** [comment]
AWS Happa migration is tracked in the separate issue, no need for this epic
